blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_2.py

```diff
@@ -0,0 +1,45 @@
+"""Formula 5.1 from NEN-EN 1992-1-1+C2:2011: Chapter Structural Analysis."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import M
+
+
+class Form5Dot2Eccentricity(Formula):
+    """Class representing formula 5.2 for the calculation of eccentricity, ei."""
```

Comment:
e_i? of e i? how do we translate the subscript?

Reply:
🥚 
In this case I would opt for ei or e,i. But in anycase an _